docs/source/more_types.rst

```diff
@@ -985,3 +989,166 @@ and non-required keys, such as ``Movie`` above, will only be compatible with
 another TypedDict if all required keys in the other TypedDict are required keys in the
 first TypedDict, and all non-required keys of the other TypedDict are also non-required keys
 in the first TypedDict.
+
+.. _literal:
+
+Literal
+*******
+
+.. note::
+
+   Literal is an officially supported feature, but is highly experimental
+   and should be considered to be in alpha stage. It is very likely that future
+   releases of mypy will modify the behavior of Literal types, either by adding
```

Comment:
```suggestion
   releases of mypy will modify the behavior of literal types, either by adding
```

Reply:
Just to double-check, did you want me to lowercase every instance where I use "Literal" as a proper noun?

I capitalized this (and every other time I used the phrase "Literal types") because I wanted to refer specifically to the `Literal[...]` type itself. I can lowercase this if there's a good reason for it, but then we should probably lowercase that phrase every other time I use it.

Reply:
Ideally I would prefer to always use one of two: `literal` or ` ``Literal`` ` (i.e. if it is capitalized, then it is ` ``code`` `).